mappings/net/minecraft/world/gen/feature/GlowstoneBlobFeature2.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_405 net/minecraft/world/gen/feature/GlowstoneBlobFeature2
```

Comment:
It is unclear how this class is different to the first one by reading the name so I would suggest replacing the "2" suffix with a more descriptive one. And if the other class gets the cluster name change, this one should too.

Reply:
It is not different, hence the unclear "2".